src/Microsoft.IdentityModel.Tokens/Json/JsonSerializerPrimitives.cs

```diff
@@ -80,7 +80,7 @@ public static Exception CreateJsonReaderExceptionInvalidType(ref Utf8JsonReader
                     LogHelper.MarkAsNonPII(reader.BytesConsumed)));
         }
 
-        public static JsonElement CreateJsonElement(List<string> strings)
+        public static JsonElement CreateJsonElement(IList<string> strings)
```

Comment:
>IList [](http://example.com/codeflow?start=52&length=5)

can we avoid the `IList`? [https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2242#discussion_r1307517240](https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/2242#discussion_r1307517240)

Reply:
IList covers a lot of types...